week-3-practise/level-1/2-every.js

```diff
@@ -11,8 +11,16 @@
 var bush1BerryColours = ["pink", "pink", "pink", "neon", "pink", "transparent"];
 var bush2BerryColours = ["pink", "pink", "pink", "pink"];
 
-var bush1SafeToEat; //Complete this statement
-var bush2SafeToEat; //Complete this statement
+var bush1SafeToEat = bush1BerryColours.every();
```

Comment:
This exercise can be done in 2 steps:
1. Create a function that finds pink bushes
2. Use that function in the two lines that you are asked to complete. 
The rest of the exercise should not be modified.